CMakeLists.txt

```diff
@@ -75,30 +78,37 @@ set(CMAKE_MODULE_PATH "${CMAKE_BINARY_DIR}/pkg/;${CMAKE_MODULE_PATH};${CMAKE_SOU
 set(PythonLibsNew_FIND_VERSION 3)
 set(Python_ADDITIONAL_VERSIONS 3.5 3.6 3.7 3.8 3.9)
 if(IVW_MODULE_PYTHON3)
-	find_package(PythonLibsNew 3 REQUIRED)
+    find_package(PythonLibsNew 3 REQUIRED)
 else()
-	find_package(PythonLibsNew 3)
+    find_package(PythonLibsNew 3)
 endif()
 
 include(cmake/globalconfig.cmake)
 
 #--------------------------------------------------------------------
 # Application setup
-option(IVW_QT_APPLICATION_BASE
-    "Build base for Qt apps. Used by the Qt network editor and the Tiny Qt app" ON)
-option(IVW_QT_APPLICATION        "Build Inviwo Qt network editor application" ON)
-option(IVW_INTEGRATION_TESTS     "Build inviwo integration test" ON)
-option(IVW_TINY_GLFW_APPLICATION "Build Inviwo Tiny GLFW Application" OFF)
-option(IVW_TINY_QT_APPLICATION   "Build Inviwo Tiny QT Application" OFF)
-
-if(IVW_QT_APPLICATION AND NOT IVW_QT_APPLICATION_BASE)
-    set(IVW_QT_APPLICATION_BASE ON CACHE BOOL "Build base for qt applications. \
-            Used by Qt network editor and Qt minimal application" FORCE)
-    message(STATUS "IVW_QT_APPLICATION_BASE was set to build, due to dependency towards IVW_QT_APPLICATION")
+option(IVW_APP_QTBASE       "Build base for Qt apps. Used by the Qt network editor and the Tiny Qt app" ON)
+option(IVW_APP_INVIWO       "Build Inviwo Qt network editor application" ON)
+option(IVW_APP_MINIMAL_GLFW "Build Inviwo Tiny GLFW Application" OFF)
```

Comment:
no we need them here to enable the modules for them (ivw_enable_modules_if) The alternative could maybe be that we just asset in the app that the needed modules are enabled and then you have to manually add them.
Which is more or less what you would do for you own app at the moment, but it would make the regular use case a bit less easy...